hackclub.com.yaml

```
@@ -2686,4 +2686,9 @@ internal:
 - ttl: 1
   type: A
   value: 134.209.190.125
+
+node:
+- ttl: 1
```

```suggestion
- ttl: 600
```
TTL needs to be bumped